### PR TITLE
Chenges in new OPM report

### PR DESCRIPTION
**Pull request by lwyszomi**
@sravfeyn @esoergel 

http://manage.dimagi.com/default.asp?176840

### CODE REVIEW

File: custom/opm/reports.py [hunk 1]
```diff
@@ -1345,3 +1345,23 @@ class LongitudinalCMRReport(MetReport):
...
+    @property
+    def redis_key(self):
+        redis_key = self.cache_key + "_" + self.slug
+        redis_key += "?blocks=%s&gps=%s&awcs=%s&is_open=%s" % (
+            self.blocks,
+            self.gp,
+            self.awcs,
+            self.request_params.get('is_open')
+        )
+        return redis_key
```
Comment (dimagimon) [inline custom/opm/reports.py]: W292 no newline at end of file

File: custom/opm/beneficiary.py [hunk 2]
```diff
@@ -48,6 +48,12 @@
...
 VHND_NO = 'VHND_no.png'
 
+BONUS = {
+    0: EMPTY_FIELD,
+    2: TWO_YEAR_AMT,
+    3: THREE_YEAR_AMT,
+}
+
 
 class OPMCaseRow(object):
```
Comment (sravfeyn) [inline custom/opm/beneficiary.py]: Does this affect other OPM reports as well (not just LCMR)?

File: custom/opm/beneficiary.py [hunk 3]
```diff
@@ -1024,10 +1048,7 @@ def __init__(self, case, report, child_index=1, **kwargs):
         self.close_mother_mo = self.get_value_from_form(CLOSE_FORM, 'form/close_mother_mo')
         self.leave_program = self.get_value_from_form(CLOSE_FORM, 'form/leave_program')
         self.close_mother_dead = self.get_value_from_form(CLOSE_FORM, 'form/close_mother_dead')
-        # TODO Add correct values
-        self.weight_this_month = EMPTY_FIELD
-        self.nutritional_status_this_month = EMPTY_FIELD
-        self.nutritional_status_bonus = EMPTY_FIELD
+        self.disp_month = datetime.date(year=1900, month=self.month, day=1).strftime("%B")
```
Comment (sravfeyn) [inline custom/opm/beneficiary.py]: Would be good to add a comment here about `year=1900`.

File: custom/opm/beneficiary.py (hunk 2 above)
Comment (lwyszomi) [inline custom/opm/beneficiary.py]: This is only in LCMR

**Comment by lwyszomi** [comment]
@sravfeyn Updated

**Pull request merged**